docs/mp/guides/migration_3x.adoc

```diff
@@ -0,0 +1,172 @@
+
+==== JakartaEE specifications (partially supported)
+
+* *CDI (Jakarta Contexts and Dependency Injection) {version-lib-jakarta-cdi}*:
++
+Changes described in link:https://jakarta.ee/specifications/cdi/3.0/jakarta-cdi-spec-3.0.html#architecture[CDI (Jakarta Contexts and Dependency Injection) {version-lib-jakarta-cdi} Specification]
+
+* *JAX-RS (Jakarta RESTful Web Services) {version-lib-jakarta-jaxrs-api}*:
++
+Moved to `jakarta` namespace. Changes described in link:https://download.eclipse.org/microprofile/microprofile-rest-client-3.0/microprofile-rest-client-spec-3.0.html#_incompatible_changes[JAX-RS (Jakarta RESTful Web Services) {version-lib-jakarta-jaxrs-api}Specification]
+
+* *JSON-B (Jakarta JSON Binding) {version-lib-jakarta-jsonb-api}*:
++
+Moved to `jakarta` namespace. Changes described in link:https://jakarta.ee/specifications/jsonb/2.0/jakarta-jsonb-spec-2.0.html#change-log[JSON-B (Jakarta JSON Binding) {version-lib-jakarta-jsonb-api} Specification]
+
+
+* *JSON-P (Jakarta JSON Processing) {version-lib-jakarta-jsonp-api}*:
++
+Moved to `jakarta` namespace.
+
+* *Jakarta Annotations {version-lib-jakarta-annotations-api}*:
++
+Moved to `jakarta` namespace. Moved to `jakarta` namespace. Full information in link:https://jakarta.ee/specifications/annotations/2.0/annotations-spec-2.0.html[Jakarta Annotations {version-lib-jakarta-annotations-api} Specification]
+
+* *Jakarta Persistence API {version-lib-jakarta-persistence-api}*:
++
+Moved to `jakarta` namespace. Changes described in link:https://jakarta.ee/specifications/persistence/3.0/jakarta-persistence-spec-3.0.html#revision-history[Jakarta Persistence API {version-lib-jakarta-persistence-api} Specification]
+
+* *Jakarta Transactions API {version-lib-jakarta-transaction-api}*:
++
+Moved to `jakarta` namespace. Changes described in link:https://jakarta.ee/specifications/transactions/2.0/jakarta-transactions-spec-2.0.html#revision-history[Jakarta Transactions API {version-lib-jakarta-transaction-api} Specification]
+
+* *Jakarta WebSocket API {version-lib-jakarta-websockets-api}*:
++
+Moved to `jakarta` namespace. Changes described in link:https://jakarta.ee/specifications/websocket/2.0/websocket-spec-2.0.html#changes[Jakarta WebSocket API {version-lib-jakarta-websockets-api} Specification]
+
+* *Jakarta Bean Validation {version-lib-jakarta-bean-validation}*:
++
+Moved to `jakarta` namespace. Changes described in link:https://jakarta.ee/specifications/bean-validation/2.0/bean-validation_2.0.html#changelog[Jakarta Bean Validation {version-lib-jakarta-bean-validation} Specification]
+
+NOTE: Please, read each specification carefully for incompatible changes!
+
+== Updated CLI
+In version 3.x Helidon CLI brings all benefits of the Project Starter to the command line allowing users to enjoy customizable project generation without leaving the terminal. See it in action xref:{rootdir}/about/cli.adoc[here]!
+
+== Project Starter
```

Comment:
Project Starter is not relevant to this guide. Remove.

Reply:
removed